app/assets/javascripts/student_profile/record_service.js

```diff
@@ -196,8 +197,7 @@
           style: styles.cancelRecordServiceButton,
           onClick: this.onClickCancel
         }, 'Cancel'),
-        (this.props.requestState === 'pending') ? dom.span({}, 'Saving...') : null,
-        (this.props.requestState === 'error') ? dom.span({}, 'Try again!') : null
+        (this.props.requestState === 'pending') ? dom.span({}, 'Saving...') : this.props.requestState
```

Comment:
As far as I can tell, we've got pending and error, which in this case I replaced with that message. Does seem kind of overloaded, though, doesn't it? An error message isn't an error state. I'll keep playing around with that. We're in the pending state for less than a second, at least on my machine. I kinda like that I hit the button and it flashes "Saving..." before displaying the new service. Makes the UI feel a bit more dynamic.